utils/test_helper.py

```diff
@@ -0,0 +1,183 @@
+from django.test import Client
+
+import os
+import random
+import string
+import filecmp
+import shutil
+
+from problem.models import Problem, Tag, Testcase, Submission
+from users.models import User
+from utils import config_info
+
+TESTCASE_PATH = config_info.get_config('path', 'testcase_path')
+TEST_PATH = ""
+
```

Comment:
About the question that whether to move `test_helper.py` to `test/utils/helper.py` or not, I would like to move it to `test/utils/helper.py` if we are going to move all the files associate with unit test to directory `test/`.
By default, each unit test file for django to run should be under the directory of the app ready to be tested. (e.g. The unit test files for testing app `problem` should be under the directory `problem/` by default.)
Therefore, if we are not going to move all the unit test files into `test/`, then the directory `test/` will contains only one file `test/utils/helper.py`. I think it is not necessary to create a new directory for just only one file.
According to documentation of django, however, it is possible for us to define our own test runner, thus maybe we can move all the unit test files into directory `test/`. But I'm not sure whether this will work indeed or not. Of course I am glad to spend some time researching about this.

The above is my opinion about the question, may I ask for the reason why Bruce prefers to move `test_helper.py` into `test/utils/helper.py`?